utils/compress/compress.go

```diff
@@ -11,40 +11,55 @@ import (
 	"path/filepath"
 	"strings"
 
-	"github.com/docker/docker/pkg/system"
-
 	"github.com/alibaba/sealer/common"
 	"github.com/alibaba/sealer/utils"
 )
 
+func validatePath(paths []string) error {
```

Comment:
this should be a primitive function definition. So I would suggest that define this as `func validatePath(path string) error` instead. When validating multiple paths necessary, you could use a loop in the calling place.